utils/mail_service.py

```diff
@@ -0,0 +1,43 @@
+import smtplib
+import ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from urllib.parse import urlencode
+from starlette.responses import JSONResponse
+from core.config import settings
+
+async def send_email(email: str, reset_token: str) -> JSONResponse:
+    """
+    Click on manage google account
+    Go to Security
+    Enable Two-Step Verification(how to sign in with google)
+    Create an App Password
+    Generate an App Password
+    """
+    email_sender = 'your-email@gmail.com'
+    email_password = 'app password not gmail password'
+    email_receiver = f'{email}'
+    reset_password_url = f"{settings.BASE_URL}reset-password?{urlencode({'token': reset_token})}"
+
+    subject = "Reset Password"
+    html = f"""\
+                <html>
+                    <body>
+                        <p>Click on this link to reset your password:</p>
+                        <p><a href="{reset_password_url}">{reset_password_url}</a></p>
+                    </body>
+                </html>
+            """
+
+    em = MIMEMultipart()
+    em['From'] = email_sender
+    em['To'] = email_receiver
+    em['Subject'] = subject
+    html_part = MIMEText(html, 'html')
+    em.attach(html_part)
+
+    context = ssl.create_default_context()
+
+    with smtplib.SMTP_SSL('smtp.gmail.com', 465, context=context) as smtp:
```

Comment:
Can we set the host and port as variables too? I think it will be better in case we want to change from gmail to zoho or any other. We don't have to change code